.travis.yml

```diff
@@ -1,4 +1,9 @@
 language: ruby
+cache: bundler
+sudo: false
 rvm:
-  - 2.1.0
-before_install: gem install bundler -v 1.11.2
+- 2.4.0
+- 2.3.1
+- 2.2.2
+
+source_key: "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcEFJQkFBS0NBUUVBMzlGeDU0WTNPa0JBaEdIcmhRNHdsdjErN2J0dmsxNDhVVm9tM29OTGpINmZtRS9nCktnZWc2TTFrRlozMnpnbkFFdTdKdW5FOTVtRWhtWW1kVDI0NmVtR1hUdWVCOFBQZE9rbGhSVlA5NXp5YzI3amMKQWdLYlVjalN1ZGxBQVFVdHk3bkR4Wmk1ejhjNTJ6SDRDRlJpSXRlbm15eldITGF5VTc2S3Z3TmdWbnk4dVM0QgplWWVDTGswRjJEaE1wbkJJQlN1RDdobWFjVm1yKy9hWW91eXQwY1hkRVd2dlNuNDkxcTBYb0FsQzUyd1dVZ2t4CjBwbkRBV0x2Yi9FL1FXb2JCNjczOTlOQkpUVlhFcmVGZ2szTFp4OGc5SmtGaFhWL0x5K3JibUNoSHk1NHcrTS8Ka3JwaUR4TTR4aFVRWi9COHpKY0FTS1NRS09EdmYweVcwRUZjZFFJREFRQUJBb0lCQUh5bkZaMHdyd21PUjJCegplTXg5L3FuWlBSd0NURDN6T2Z6RkRPallqdmNNZnE1QnJrTTZSRy9rRkYzRndITTRFdlY0Y0htYUlETnkzVmU1CndSYy8rT20zK0tGRlpxVmY3RkpPRkJLY21NaDBoWmdKK041VnZWTnJXWkdRWVlrdjdUV05uRzhzaERBb2NSeVgKVStDdnZvaFJPMU85UVBKMC9oU09CcEFqdlQvczVpWHB1bkczMlQyNHgxeVo5cWtHbm5zcG9lT0hQd0Evb2poeApCeTkzd1RnQUh0OTA4dk52MEcrLzVTaVhNQjcxYmhGcGhFanpGOVZ2UkpMZ2MzR0E5RGswSW1pRTdxTVI5U1c5CnMxd1lGTjZjRXdUSDhXQW1JbFRieGNJS3R3VWE0OGhXeENib1RLRFMxK0pUSkg5SlRHSnErNWRrNzIyYm1YQ24KOVlhMjhvRUNnWUVBOWJzMmFtSWUrUzFZZG1QdkIxRlMxNUlPSExLYTdWeTBDUlkyV2NoZ1BaZnJFdnV4RGZQQwpDSjdKdnFPb2hOY09MVnBzeUxSMm5ERUdnZjlNTTB0QkNJMTBtSnJSeE5PYW9FY01ST2syTmMwQnV0M0lTTkp1CmZmNUV1MTlPVHQ0TG8wUGJVaTdCNUgrSlVFZHVLbzMvR0lvVVdnVSttbmtCWGRQZEliVUlYNTBDZ1lFQTZTdlAKUHJIVmVxOU5PK1c3QXBqaTY2QXl0NjkzSThHSHZacm44aGVobld5S1R0OEdQeCtQWUpEblV6UUNGS0Evdm5HOApoWDNydGJpcE9qbVlFYkwzWnpBVG9hNGxGOU9UK3JmbC9jbDJaeC9ZYmIzNUh2SlNpZS90bGFkU1Zid0gzZnBqCldyeUNLSnBYOEFpRklNNDBGSWV5cWtsWG1wcHlQd1M1VnFYc0ZMa0NnWUVBM3dWQ3Bkclg4WjdPSXVVZ0U2d2oKVzdNYlM4QnVDKytQRnBBcVUrSWdOc2FpZTJBam1WNGFSR2laQVlUWGRYa1pUT0E4eXd1K2VwT2tMR2ZnYU0xcwpBSzhobXo4b0UxTlppdzN3ZkxxTWtOZTNNam9IaE5BM3AvSVZ2MWtJWVZ2MWhBQWFGY1NLQlRBYkY5azhpekxFCnl5akd2OHp5dkpBR1JYbjd3SlBsUGFFQ2dZQXdYMkpUUTFsaVpsUERoVmxTcEowTy9PRUlzbUlwRE5FNFNKYXQKNXJFa2s0dnQ2TjdHNGFsQkEwNjZPQ1BtVzRoZDNnS2FQeEZUVmNmbHlHVC81dnprYjFqalJkbDBJL1NqSXR1WAoxdCsxYlNuZE9uZ2dMMjFvS0ZwRm5ieTdCSk10UGZDMnRqemFaNHF1eFZXNUJJNVJaQVZ3eGw1aU9JQlRDb3ZZCnFmdGNJUUtCZ1FEYnZjUEZrMjFuRXRCZ1gxeHUyOVZSTjJtUlQrTFZ4OWVjK3c2Y25QV2Q1NU5HdWlVUnBrQWEKaW0vY040bHNUTlpGR0MvZmpBWWZBdEpVTHViL0JQRTdPY3lwOVFIYjVsRnRibklyRDhHREFtcW1SZWdrNzgxeQpFWmRPMUZCZ2JScFJzWTVJM1JTbEFRRmlrZmlYL2c1aHlmaG9PTDhKYk1obGRpZmE2UU14SUE9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo="
```

Comment:
hey @martinechtner we're gonna have to yank this key since this is a public repo